templates/OME-TIFF_Conversion.ipynb

```diff
@@ -0,0 +1,160 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    load_utils.fov_to_ome(data_dir=data_dir, ome_save_dir=save_dir, img_sub_folder=img_sub_folder fovs=fovs, channels=channels)
</pre><p> Missing a comma in between "img_sub_folder" and "fovs"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/ark-analysis/pull/819/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>